html-aam/index.html

```diff
@@ -804,18 +805,35 @@ <h4 id="el-audio">`audio`</h4>
             </tr>
           </tbody>
         </table>
-        <h4 id="el-autonomous-custom-element">autonomous custom element</h4>
+        <h4 id="el-autonomous-custom-element">
+          autonomous custom elements
+          and
+          <span id="el-form-associated-custom-element">form-associated custom elements</span>
+        </h4>
         <table aria-labelledby="el-autonomous-custom-element">
           <tbody>
             <tr>
               <th>HTML Specification</th>
               <td>
                 <a data-cite="HTML">autonomous custom element</a>
+                and
+                <a data-cite="html/custom-elements.html#custom-elements-face-example">form-associated custom element</a>
               </td>
             </tr>
             <tr>
               <th>[[wai-aria-1.2]]</th>
-              <td>If the author assigned a conforming ARIA role using the `role` attribute, map to that role. Otherwise, the <a class="core-mapping" href="#role-map-generic">`generic`</a> role.</td>
+              <td>
+                <ul>
+                  <li>If the author assigned a conforming ARIA role using the `role` attribute, or by the custom element's internals: map to the specified role.</li>
+                  <li>else if the author assigned HTML attributes that result in a <a>minimum role</a>:  map to the minimum role.</li>
+                  <li>else if the custom element is focusable: map to the <a class="core-mapping" href="#role-map-generic">`group`</a> role</li>
+                  <li>else if an author specifies a global ARIA attribute on the custom element that creates a relation with another element: map to the <a class="core-mapping" href="#role-map-generic">`group`</a> role</li>
+                  <li>else if the custom element has no attached shadow root: map to the <a class="core-mapping" href="#role-map-generic">`generic`</a> role</li>
+                  <li>else if the custom element has an `aria-live` attribute: map to the <a class="core-mapping" href="#role-map-generic">`generic`</a> role</li>
```

Comment:
i'm not sure i follow this comment - since custom elements are presently generic anything that says to map this to generic is essentially keeping the status quo.